### PR TITLE
makes it so that the starting a quiz only uses one worker on canvas side

### PR DESCRIPTION
**Pull request by dittonjs**

### CODE REVIEW

File: app/controllers/api/proctored_exams_controller.rb [hunk 1]
```diff
@@ -1,77 +1,38 @@
...
+    render json: { exam_request: @exam_request }
+  end
 
-    if !matched_code
-      render(
-        json: { error: "Unauthorized: Invalid Proctor Code." },
-        status: :unauthorized,
+  def update
+    ExamRequest.find(params[:id]).
+      update_attributes(
```
Comment (bfcoder) [inline app/controllers/api/proctored_exams_controller.rb]: Use `.update` instead

File: config/routes.rb [hunk 2]
```diff
@@ -52,8 +52,8 @@
   namespace :api do
     get "proctor_login" => "proctor_login#signed_url"
     get "proctored_exams" => "proctored_exams#start_proctored_exam"
+    resources :proctored_exams
```
Comment (bfcoder) [inline config/routes.rb]: `resources :proctored_exams, only: [:update]`
Don't need extra routes that aren't used. :]

**Comment by bfcoder** [comment]
So this change enables this to use one web worker for adhesion and one for canvas. Correct?

**Comment by dittonjs** [comment]
Yup!

**Pull request merged**